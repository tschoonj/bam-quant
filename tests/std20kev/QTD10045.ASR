File: C:\xrfdaten\2014KW30\chris\std20kev\std10045.spe
Start: 21.07.2014 18:20:59
Stop: 21.07.2014 18:22:40
BESSY: 298.1464843 298.1127886
IONI: 2.952838E-11 2.938272E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 97759
Run: 100000
Dead: 2
ICR: 1421
OCR: 0
Accumulated BESSY: 59706.5826728
Accumulated Ioni: 5.90302102E-09
N Accumulation: 200
Normfactor:  3.4657634e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Rh
exp_x 91.2603988647461 91.2604
exp_y 92.3237991333008 92.3238
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  45   2      2.696     15327.       136.       2.92
