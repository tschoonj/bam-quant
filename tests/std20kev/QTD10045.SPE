$DATA:
0 2000
22601
14530
6092
1797
343
60
8
0
0
0
0
0
0
0
0
0
5
50
49
44
50
35
55
29
40
50
38
28
33
30
40
27
24
34
23
39
30
22
31
35
34
28
34
39
48
42
65
69
60
45
51
35
38
38
43
37
34
36
43
35
28
27
30
25
28
24
17
23
28
32
18
13
24
23
23
10
21
20
27
28
16
25
25
26
21
21
23
26
20
19
29
20
19
25
31
27
39
43
46
62
85
88
107
114
114
108
112
86
81
67
38
44
39
42
58
80
85
105
101
100
76
63
68
60
66
90
85
154
259
491
852
1422
2094
2497
2488
2183
1756
1404
1206
1268
1399
1480
1482
1307
1205
1030
800
601
398
276
216
162
99
92
53
61
57
49
33
33
27
21
30
37
39
62
63
41
51
47
26
19
16
14
7
11
14
14
9
14
14
22
16
15
9
7
13
11
13
9
14
19
7
10
10
3
15
13
16
11
13
11
6
13
8
5
10
14
10
10
11
14
14
17
13
10
18
13
18
3
11
14
13
14
18
17
19
15
11
15
15
20
16
8
13
21
18
17
17
14
18
18
14
16
19
13
17
19
7
18
11
17
15
18
11
19
23
18
16
19
21
15
12
20
9
23
14
20
21
25
28
16
16
16
20
18
8
17
17
11
23
17
12
12
16
11
18
23
24
17
16
13
29
17
30
13
15
19
25
13
21
15
18
28
20
21
25
21
29
40
66
110
174
273
348
509
583
582
553
464
338
236
141
98
48
34
27
21
17
21
17
17
17
13
15
17
19
22
17
18
22
21
34
36
46
51
66
89
91
96
80
71
59
57
46
28
26
32
29
23
16
25
16
25
21
18
33
35
43
31
42
34
35
39
26
21
35
24
25
26
25
25
25
22
16
24
18
22
24
36
17
28
33
39
32
57
44
64
56
65
53
49
75
71
92
101
135
142
193
180
183
157
156
112
91
80
55
50
32
28
40
32
43
43
38
46
40
45
59
61
40
43
40
40
30
26
30
37
37
33
38
33
31
43
36
36
35
43
60
68
80
84
111
119
97
134
161
215
295
414
638
902
1253
1760
2236
2668
3090
3208
3162
2970
2450
1875
1421
977
587
385
254
144
98
82
57
45
57
43
47
46
37
38
45
56
73
67
77
85
84
89
69
81
42
57
44
29
24
22
29
26
32
33
24
26
24
42
38
33
37
34
31
36
30
38
41
60
56
52
58
52
78
72
102
92
127
157
160
229
253
305
352
355
385
417
498
575
779
1025
1266
1549
1899
2112
2145
2197
2009
1847
1586
1389
1307
1144
1051
1005
904
798
686
516
406
302
229
146
114
87
87
83
77
84
94
98
73
82
64
71
66
61
52
49
45
54
38
41
32
27
32
29
23
29
25
24
25
22
23
18
20
32
18
20
17
26
40
28
25
31
31
23
24
27
34
35
26
36
41
51
44
49
56
48
42
52
37
31
36
45
35
37
45
55
72
88
121
199
233
281
308
347
376
400
356
320
270
215
184
130
100
107
87
101
89
91
109
115
123
121
140
168
140
110
122
97
97
93
68
53
34
47
41
19
27
34
27
26
31
28
23
35
37
31
33
46
49
41
40
43
32
34
31
28
32
35
22
22
25
24
23
27
19
27
26
29
32
22
27
24
28
28
23
34
33
33
24
36
26
21
35
22
32
34
25
37
32
27
24
25
31
26
21
32
26
42
41
30
32
33
35
23
25
29
22
28
20
36
23
25
34
30
33
37
33
30
39
35
37
48
34
40
40
37
27
30
43
20
30
52
30
44
53
51
45
54
68
67
72
69
72
81
76
75
84
76
74
48
66
51
50
44
43
43
33
41
41
38
28
56
34
53
48
63
44
59
65
57
65
48
56
36
49
36
43
56
58
52
63
60
45
53
53
50
54
55
43
67
56
52
63
64
53
56
52
51
43
39
47
36
34
39
49
27
27
26
21
27
20
16
20
16
22
10
13
13
11
13
9
12
3
10
7
8
7
12
11
11
15
11
12
20
10
10
14
19
18
14
8
20
15
17
26
15
10
12
10
19
10
10
17
12
15
15
16
16
16
14
11
10
21
22
15
21
18
13
13
17
17
20
28
14
26
23
36
21
27
27
26
23
26
23
35
15
26
35
23
21
33
23
31
39
32
20
26
32
34
44
28
48
37
42
45
41
41
50
44
50
44
57
54
56
53
52
63
58
74
58
72
69
54
55
52
70
68
71
54
59
64
65
74
76
72
102
100
119
155
171
204
207
289
308
347
369
395
461
461
483
495
477
481
474
490
439
433
432
438
428
442
447
385
383
452
380
331
305
277
214
175
128
118
85
54
62
36
13
10
9
4
3
4
3
1
1
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
1
0
0
0
0
1
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
3
1
0
1
1
1
4
2
3
0
1
1
1
2
0
0
3
0
0
0
0
0
0
0
1
0
1
0
0
1
1
0
1
2
0
1
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10045.spe
Start: 21.07.2014 18:20:59
Stop: 21.07.2014 18:22:40
BESSY: 298.1464843 298.1127886
IONI: 2.952838E-11 2.938272E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 97759
Run: 100000
Dead: 2
ICR: 1421
OCR: 0
Accumulated BESSY: 59706.5826728
Accumulated Ioni: 5.90302102E-09
N Accumulation: 200
Normfactor:  3.4657634e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Rh
 
exp_x 91.2603988647461 91.2604
exp_y 92.3237991333008 92.3238
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  45   2      2.696     15327.       136.       2.92
