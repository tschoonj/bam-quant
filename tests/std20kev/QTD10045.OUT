     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:47  บ   
     บ  Spectrum: QTD10045.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        4 iterations done    บ   
     บ  ChiSquare =    57.6   last change =   .06%    lambda= 1.E-04   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -67.6 ๑   100.0   บ     -67.6 ๑     6.7   บ   
     บ  GAIN (eV/ch)  บ     20.541 ๑   2.000   บ    20.541 ๑    .050   บ   
     บ det NOISE (eV) บ       56.5 ๑    40.0   บ      56.5 ๑    30.3   บ   
     บ  FANO factor   บ       .199 ๑    .050   บ      .199 ๑    .041   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Rh-L1                              3368. ๑     89.           บ   
     บ    L1M3        2.916    .51082        1721. ๑     47.     44.5  บ   
     บ              145.248    124.74         161.           4.34E-01  บ   
     บ    L1M2        2.892    .29794        1004. ๑     29.     43.7  บ   
     บ              144.079    124.33         161.           4.25E-01  บ   
     บ    L1N3        3.364    .19124         644. ๑     21.     30.8  บ   
     บ              167.057    132.14         153.           5.66E-01  บ   
     บ                                                                 บ   
     บ  2 Rh-L2                              7644. ๑    125.           บ   
     บ    L2M4        2.834    .88581        6771. ๑    112.     34.0  บ   
     บ              141.256    123.34         162.           4.03E-01  บ   
     บ    L2N4        3.144    .08938         683. ๑     17.     48.2  บ   
     บ              156.347    128.56         157.           5.11E-01  บ   
     บ    L2M1        2.519    .01886         144. ๑     12.     11.0  บ   
     บ              125.921    117.79         140.           2.85E-01  บ   
     บ    L2N1        3.065    .00594          45. ๑     13.     55.0  บ   
     บ              152.501    127.25         158.           4.85E-01  บ   
     บ                                                                 บ   
     บ  3 Rh-L3                             17379. ๑    154.           บ   
     บ    L3M5        2.696    .88188       15327. ๑    136.      2.9  บ   
     บ              134.538    120.94         165.           3.57E-01  บ   
     บ    L3N5        3.001    .09006        1565. ๑     19.     49.7  บ   
     บ              149.386    126.18         159.           4.64E-01  บ   
     บ    L3M1        2.377    .02068         359. ๑     12.     12.4  บ   
     บ              119.008    115.21         142.           2.27E-01  บ   
     บ    L3N1        2.923    .00738         128. ๑     13.     44.5  บ   
     บ              145.589    124.86         161.           4.36E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
